binary/util.py

```diff
@@ -231,6 +231,7 @@ def __init__(self, encoder):
             'List_Integer': partial(FixSizedParamEncoder.encode_fix_sized_list_frame, item_type='int'),
             'List_Long': partial(FixSizedParamEncoder.encode_fix_sized_list_frame, item_type='long'),
             'List_UUID': partial(FixSizedParamEncoder.encode_fix_sized_list_frame, item_type='UUID'),
+            'List_Data': partial(VarSizedParamEncoder.encode_data_frame),
```

Comment:
This should be `'List_Data': partial(self.encode_multi_frame_list, encoder=self.encode_data_frame),` . Right now it is encoding a data frame instead of the list of data frames. That is the cause of the failing test

Reply:
thanks @mdumandag ! i had an inkling it was the binary encoding but not sure what